RoT/setup-RoT.tp2

```diff
@@ -53,7 +53,7 @@ ALWAYS
 			    "schinese" => "CP936"
 			END
 			ACTION_DEFINE_ARRAY charsetsReloadArray BEGIN setup scripts journal END
-			ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN setup-ee END
+			ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN setup setup-ee END
```

Comment:
Hey, this is incorrect.  The `setup.tra` file has to be converted for EE engine, as it is encoded in local code pages.